CONTRIBUTING.md

```diff
@@ -24,6 +36,26 @@ It is encouraged to set up such a hook.
 The script may be called with the `-b` option to include compilation in the
 hook.
 
+About code, **remember to break the line after `=`, if it is too long**.
+You can find more details in the previous sections.
+
+Importing of librairies
+-----------------------
+
+Import a library when you use it many times.
+But to avoid problems with generic names (make, get, ...), you have to put the name of the library before the name of the function.
+
+Naming of variables
+-------------------
+
+Type of the variable | Name of the variable
+-------------------- | --------------------
+       prop          |        p
+       expo          |        e
+       term          |        t
+       sym           |
```

Comment:
There's nothing for sym. But is it useful to go into these details? Moreover, a same name can be used with different types. p is used for prop but also for positions, s is used for sym but also for string, etc.

Reply:
Yes, it's a good question! That's why I left a blanck.
This part isn't finished but I'm not sure it's easy to read and very useful.

Reply:
OK but naming schemes could be useful? I'm not sure!

Reply:
We should not impose too many constraints and keep things simple and easy to adopt.

Reply:
I have removed it.